### PR TITLE
Add udev rules and build script for upboard 4000 boards

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

File: hardware/system/upboard/gaia/udev/99-gaia.rules [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-KERNELS=="1-7.3:1.0", SYMLINK+="ttyJ5"
+KERNELS=="1-7.2:1.0", SYMLINK+="ttyUSB0"
+KERNELS=="1-7.3:1.0", SYMLINK+="ttyUSB1"
 KERNEL=="ttyS1", SYMLINK+="ttyJ7"
 KERNEL=="video[0-9]*", ATTR{name}="USB 2.0 Camera", ATTRS{idVendor}=="05a3", ATTRS{idProduct}=="8830", SYMLINK+="video-scan"
 KERNEL=="video[0-9]*", ATTR{name}="5M Camera ", ATTRS{idVendor}=="090c", ATTRS{idProduct}=="f37d", SYMLINK+="video-front"
-SUBSYSTEM=="video4linux", KERNELS=="1-2", ATTRS{idVendor}=="0ac8", ATTRS{idProduct}=="0345", SYMLINK+="video-scan"
-SUBSYSTEM=="video4linux", KERNELS=="1-3", ATTRS{idVendor}=="0ac8", ATTRS{idProduct}=="0345", SYMLINK+="video-front"
+SUBSYSTEM=="video4linux", KERNELS=="1-5", ATTRS{idVendor}=="0ac8", ATTRS{idProduct}=="0345", SYMLINK+="video-scan"
+SUBSYSTEM=="video4linux", KERNELS=="1-6", ATTRS{idVendor}=="0ac8", ATTRS{idProduct}=="0345", SYMLINK+="video-front"
```
Comment (siiky) [inline hardware/system/upboard/gaia/udev/99-gaia.rules]: Aren't these only for the new UP 4000? Machines using the old board should stay the same right?

File: hardware/system/upboard/sintra/udev/99-sintra.rules [hunk 2]
```diff
@@ -1,5 +1,5 @@
-KERNELS=="1-7.2:1.0", SYMLINK+="ttyJ4"
-KERNELS=="1-7.3:1.0", SYMLINK+="ttyJ5"
+KERNELS=="1-7.2:1.0", SYMLINK+="ttyUSB0"
+KERNELS=="1-7.3:1.0", SYMLINK+="ttyUSB1"
 KERNEL=="ttyS1", SYMLINK+="ttyJ7"
-SUBSYSTEM=="video4linux", KERNELS=="1-2", ATTRS{idVendor}=="0ac8", ATTRS{idProduct}=="0345", SYMLINK+="video-scan"
-SUBSYSTEM=="video4linux", KERNELS=="1-3", ATTRS{idVendor}=="0ac8", ATTRS{idProduct}=="0345", SYMLINK+="video-front"
+SUBSYSTEM=="video4linux", KERNELS=="1-5", ATTRS{idVendor}=="0ac8", ATTRS{idProduct}=="0345", SYMLINK+="video-scan"
+SUBSYSTEM=="video4linux", KERNELS=="1-6", ATTRS{idVendor}=="0ac8", ATTRS{idProduct}=="0345", SYMLINK+="video-front"
```
Comment (siiky) [inline hardware/system/upboard/sintra/udev/99-sintra.rules]: Same here

File: hardware/system/upboard-4000/install_script.sh [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+if [ "$PRINTER" == "zebra" ]; then
+  PRINTER='Zebra-KR-403'
+fi
+
+# Cert workaround
+cat > 99-lamassu << EOL
+Acquire::https::ubilinux.org::Verify-Peer "false";
+Acquire::https::ubilinux.org::Verify-Host "false";
+EOL
```
Comment (siiky) [inline hardware/system/upboard-4000/install_script.sh:42]: Hmmm is this necessary? I think we're gonna start using plain Ubuntu and Ubuntu's servers/mirrors on the UP 4000 (as recommended by them).

**Review by siiky — CHANGES_REQUESTED** [review]

File: hardware/system/upboard/gaia/udev/99-gaia.rules (hunk 1 above)
Comment (chaotixkilla) [reply hardware/system/upboard/gaia/udev/99-gaia.rules]: Yeah, the files ended up switched between regular upboard and upboard-4000 directories. Should be fixed

File: hardware/system/upboard-4000/install_script.sh (hunk 3 above)
Comment (chaotixkilla) [reply hardware/system/upboard-4000/install_script.sh:42]: Rafael said this is currently a pending issue and to leave it as is, for now

**Review by siiky — APPROVED** [review]

**Comment by siiky** [comment]
@RafaelTaranto Replaced by #1014, can be closed.

**Pull request closed without merging**